packages/astro/src/transform/imports.ts

```diff
@@ -0,0 +1,36 @@
+import { t, visit } from './babel.js';
+
+export function ensureImport(root: t.File, importDeclaration: t.ImportDeclaration) {
+	let specifiersToFind = [...importDeclaration.specifiers];
+
+	visit(root, {
+		ImportDeclaration(path) {
+			if (path.node.source.value === importDeclaration.source.value) {
+				path.node.specifiers.forEach((specifier) =>
+					specifiersToFind.forEach((specifierToFind, i) => {
+						if (specifier.type !== specifierToFind.type) return;
+						if (specifier.local.name === specifierToFind.local.name) {
+							specifiersToFind.splice(i, 1);
+						}
+					})
+				);
+			}
+		},
+	});
+
+	if (specifiersToFind.length === 0) return;
+
+	visit(root, {
+		Program(path) {
+			const declaration = t.importDeclaration(specifiersToFind, importDeclaration.source);
+			const latestImport = path
+				.get('body')
+				.filter((statement) => statement.isImportDeclaration())
+				.pop();
+
+			// It's inserted before because of formatting issues
```

Comment:
just checking if this is still the case, feel free to resolve if so

Reply:
Yes, unfortunately `insertAfter` has a tendency to separate comments from the nearest node and create noisy diffs.